go.mod

```diff
@@ -27,10 +27,10 @@ require (
 	gomodules.xyz/jsonpatch/v2 v2.4.0
 	google.golang.org/api v0.151.0
 	google.golang.org/protobuf v1.33.0
-	istio.io/api v1.19.4
+	istio.io/api v1.21.5
 	istio.io/client-go v1.19.4
-	k8s.io/api v0.28.4
-	k8s.io/apimachinery v0.28.4
+	k8s.io/api v0.29.0
+	k8s.io/apimachinery v0.29.0
```

Comment:
Is it intended to bump these versions as well?

Reply:
I believe these automatically upgraded when running go mod tidy? After upgrading the istio libs

Reply:
@terrytangyuan, correct, see that we are pinning the k8s version here:
https://github.com/kserve/kserve/pull/3825/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R141

there were some incompatibilities with this new version that needs closer attention.